submissions/kotlyar-andrey/a-tiny-JS-world/index.js

```diff
@@ -0,0 +1,49 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/kotlyar-andrey/a-tiny-JS-world
+   Web app: https://kotlyar-andrey.github.io/a-tiny-JS-world/
+   */
+
+function Creature(species, name, gender, legs, hands, saying) {
+  this.species = species;
+  this.name = name;
+  this.gender = gender;
+  this.legs = legs;
+  this.hands = hands;
+  this.saying = saying;
+
+  this.properties = [
+    "species",
+    "name",
+    "gender",
+    "legs",
+    "hands",
+    "saying",
+    "friendsNames",
+  ];
```

Comment:
Properties that aren't intended for direct external (outside class definition) access should be declared as private. This is achieved by prepending their names with `_` (conventional) or `#` (class syntax). The latter is not applicable here.